templates/agenda/show.html.twig

```diff
@@ -82,9 +86,11 @@
                 <div class="col">
                     <h5 id="section_agenda_items">{% trans %}Agenda items{% endtrans %}</h5>
                 </div>
-                <div class="col-auto">
-                    <a class="btn btn-success btn-sm {% if agenda.isFinished() %} disabled {% endif %}" href="{{ url('agenda_item_create', {id: agenda.id}) }}" role="button">{% trans %}Add agenda item{% endtrans %}</a>
-                </div>
+                {% if not is_granted('ROLE_BOARD_MEMBER') %}
```

Comment:
Use `{% if is_granted('ROLE_CASEWORKER') %}`

Reply:
Fixed in 4bc1a7723936a2511ae7d457d4dfec56d37c7bf1.